src/EditorFeatures/Core/Implementation/CodeCleanup/CodeCleanupService.cs

```diff
@@ -0,0 +1,165 @@
+            //IDEDiagnosticIds.PreferIntrinsicPredefinedTypeInDeclarationsDiagnosticId,
+            //    IDEDiagnosticIds.PreferIntrinsicPredefinedTypeInMemberAccessDiagnosticId,
+            //    IDEDiagnosticIds.InlineAsTypeCheckId,
+            //    IDEDiagnosticIds.InlineIsTypeCheckId,
+            //    IDEDiagnosticIds.InlineIsTypeWithoutNameCheckDiagnosticsId,
+
+            return dictionary.ToImmutableDictionary();
+        }
+
+        public Task<IEnumerable<TextChange>> GetChangesForCleanupDocument(Document document, CancellationToken cancellationToken)
+        {
+            var oldDocument = document;
+            document = RemoveSortUsings(document, cancellationToken);
+            document = ApplyCodeFixes(document, cancellationToken);
+
+            return document.GetTextChangesAsync(oldDocument, cancellationToken);
+        }
+        private Document RemoveSortUsings(Document document, CancellationToken cancellationToken)
+        {
+            // remove and sort usings
+            if (document.Project.Solution.Workspace.Options.GetOption(FeatureOnOffOptions.RemoveUnusedUsings, LanguageNames.CSharp))
+            {
+                var removeUsingsService = document.GetLanguageService<IRemoveUnnecessaryImportsService>();
+                if (removeUsingsService != null)
+                {
+                    document = removeUsingsService.RemoveUnnecessaryImportsAsync(document, cancellationToken).WaitAndGetResult(cancellationToken);
+                }
+            }
+
+            // sort usings
+            if (document.Project.Solution.Workspace.Options.GetOption(FeatureOnOffOptions.SortUsings, LanguageNames.CSharp))
+            {
+                document = OrganizeImportsService.OrganizeImportsAsync(document, cancellationToken).WaitAndGetResult(cancellationToken);
+            }
+
+            return document;
+        }
+
+        private Document ApplyCodeFixes(Document document, CancellationToken cancellationToken)
+        {
+            var fixAllService = document.Project.Solution.Workspace.Services.GetService<IFixAllGetFixesService>();
+
+            var dummy = new ProgressTracker();
+            foreach (var diagnosticId in GetEnabledDiagnosticIds(document.Project.Solution.Workspace))
+            {
+                var length = document.GetSyntaxTreeAsync(cancellationToken).WaitAndGetResult(cancellationToken).Length;
+                var textSpan = new TextSpan(0, length);
+
+                var fixCollection = _codeFixService.GetFixesAsync(document, textSpan, diagnosticId, cancellationToken).WaitAndGetResult(cancellationToken);
+                if (fixCollection == null)
+                {
+                    continue;
+                }
+
+                var fixAll = fixCollection.FixAllState;
+                var solution = fixAllService.GetFixAllChangedSolutionAsync(fixAll.CreateFixAllContext(dummy, cancellationToken)).WaitAndGetResult(cancellationToken);
```

Comment:
these lines are too long.  try to keep lines from going off hte screen on github.  Thanks!

Reply:
fixed